lib/lotus/cli.rb

```diff
@@ -79,6 +79,19 @@ def new(name = nil)
       end
     end
 
+    desc 'generate', 'generates a new action'
+    method_option :path,                desc: 'applications path',        type: :string, default: 'apps'
```

Comment:
@joneslee85 This is to describe where applications are, located. It defaults to `apps/`. But yeah, it doesn't look important.